metadata.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "ghoneycutt-hosts",
-  "version": "2.2.3",
+  "version": "2.2.4",
```

Comment:
You should never bump versions in your pull requests for other software. I might merge another PR before this one, then the versions would not be right and I would have to rebase. When people merge they will handle the release process.